pallets/pallet-did-lookup/Cargo.toml

```diff
@@ -55,6 +60,7 @@ std = [
   "scale-info/std",
   "sp-runtime/std",
   "sp-std/std",
+	"impl-serde/std",
```

Comment:
Don't we need to add other crates with `std` feature as well?

Reply:
I think we need atleast `sp-io` and `sp-core` here